jeni.py

```diff
@@ -399,6 +398,12 @@ class Injector(BaseInjector):
 
             with Injector() as injector:
                 injector.get('injector')
+
+        Local values can be provided when constructing the injector::
+
+            with RequestInjector(request=request) as injector:
+                injector.get('form:username')
```

Comment:
Maybe include both? Start with the trivial, then mention that a useful pattern is in creating request lifecycles.

This allows values to be provided trivially:

```python
with Injector(foo='bar') as injector:
    injector.get('foo') # 'bar'
```

A useful pattern is to have the injector's providers key off input data:

```python
with RequestInjector(request=request) as injector:
    injector.get('form:username')
```